recipes-containers/docker/docker-ce_git.bbappend

```diff
@@ -3,6 +3,7 @@ FILESEXTRAPATHS:prepend := "${THISDIR}/${BPN}:"
 SRC_URI:append = " \
     file://0001-dockerd-daemon-use-default-system-config-when-none-i.patch \
     file://0002-cli-config-support-default-system-config.patch \
+    file://0001-Correct-panic-when-IPv4-lacks-IFA_ADDRESS-address.patch \
```

Comment:
Very minor thing, if you could fix the patch numbering so it increments. (i.e. 0003*)

Reply:
Sure, already updated.